TodoApp/src/TodoApp.Api/Controllers/TodosController.cs

```diff
@@ -10,16 +10,14 @@
 namespace TodoApp.Api.Controllers
 {
     [ApiVersion("1.0")]
-    [RoutePrefix("api/v{version:apiVersion}/todos")]
-    [Route("{id:guid?}", Name = DEFAULT_ROUTE)]
+    [Route("api/v{version:apiVersion}/todos/{id:guid?}", Name = DEFAULT_ROUTE)]
     public class TodosController : ApiController
     {
+        public const string DEFAULT_ROUTE = "TodosDefault";
+
         private readonly ITodoRepository _repository;
         private readonly IUriHelper _uriHelper;
 
-        public const string DEFAULT_ROUTE = "Default";
-        public const string POST_TODO_ROUTE = "PostTodo";
```

Comment:
The constant is still used in `UriHelperTests`. You'll need to get rid of that usage.